projects/js-packages/publicize-components/changelog/add-social-image-generator-feature-flag

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: added
+
+SIG feature check
```

Comment:
Do you think we can make this a bit more descriptive? I think it might end up in the changelog for Jetpack and Jetpack Social.